That osx error looks weird. I might queue up another job.

```
Mode                LastWriteTime         Length Name                                                                  
----                -------------         ------ ----                                                                  
d-----        6/11/2019   4:52 AM                raw xml results                                                       
current job: 2367
baseline job: 2343
baseline commitid: d962dc7d651b563bed089f4bf9056003ebb7fe1c
merge baseline: 143ae13d28063ea404e75692083db26e02d36c66

baseline does not line up with merge, attempting to adjust to new baseline.  This is typically caused the post checkin CI tests getting backed up.
##[error] Missing log files for comparison baseline (d962dc7d651b563bed089f4bf9056003ebb7fe1c).  This may cause unrelated failures to show up in the test results. This is most likely the result of an infrastructure failure in the post checkin pipeline which creates the baseline logs. If this is the case, the fix is to manually queue a post checkin CI job (use default settings) and rerun this job when it completes.
-----------------------------------------------
##[error]Missing x64-osx test results in current test run

x64-osx results are inconclusive because it is missing logs from test


% : Cannot index into a null array.
At C:\vsts\_work\3\s\analyzeTestResults.ps1:721 char:87
+ ... r the following triplets: $($hash.Keys | %{"$($_)($($hash[$_]))"})`n"
+                                              ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [ForEach-Object], RuntimeException
    + FullyQualifiedErrorId : NullArray,Microsoft.PowerShell.Commands.ForEachObjectCommand
 
##[error]PowerShell exited with code '1'.
##[section]Finishing: Analyze results and prepare test logs
```

--- a/ports/ixwebsocket/CONTROL
+++ b/ports/ixwebsocket/CONTROL
@@ -5,14 +5,17 @@ Description: Lightweight WebSocket Client and Server + HTTP Client
 Default-Features: ssl
 
 Feature: ssl
-Build-Depends: ixwebsocket[openssl] (!windows&!osx), ixwebsocket[mbedtls] (windows)
+Build-Depends: ixwebsocket[openssl] (!windows&!osx), ixwebsocket[mbedtls] (windows), ixwebsocket[sectransp] (osx)
 Description: Default SSL backend
 
 # SSL backends
+Feature: openssl
+Build-Depends: openssl
+Description: SSL support (OpenSSL)
+
 Feature: mbedtls
 Build-Depends: mbedtls
 Description: SSL support (mbedTLS)
 
-Feature: openssl
-Build-Depends: openssl
-Description: SSL support (OpenSSL)
+Feature: sectransp
+Description: SSL support (sectransp)

--- a/ports/ixwebsocket/CONTROL
+++ b/ports/ixwebsocket/CONTROL
@@ -1,21 +1,4 @@
 Source: ixwebsocket
 Version: 4.0.3
-Build-Depends: zlib
+Build-Depends: zlib, openssl, mbedtls
 Description: Lightweight WebSocket Client and Server + HTTP Client
-Default-Features: ssl
-
-Feature: ssl
-Build-Depends: ixwebsocket[openssl] (!windows&!osx), ixwebsocket[mbedtls] (windows), ixwebsocket[sectransp] (osx)
-Description: Default SSL backend
-
-# SSL backends
-Feature: openssl
-Build-Depends: openssl
-Description: SSL support (OpenSSL)
-
-Feature: mbedtls
-Build-Depends: mbedtls
-Description: SSL support (mbedTLS)
-
-Feature: sectransp
-Description: SSL support (sectransp)

--- a/ports/ixwebsocket/portfile.cmake
+++ b/ports/ixwebsocket/portfile.cmake
@@ -11,47 +11,16 @@ vcpkg_extract_source_archive_ex(
     ARCHIVE ${ARCHIVE} 
 )
 
-# ws exe
-set(USE_WS OFF)
-if("tool" IN_LIST FEATURES)
-    set(USE_WS ON)
-endif()
-
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA # Disable this option if project cannot be built with Ninja
-    OPTIONS
-    	-DUSE_TLS=1
-	-DUSE_WS=1
+    OPTIONS -DUSE_TLS=1
 )
 
 vcpkg_install_cmake()
-
-# the native CMAKE_EXECUTABLE_SUFFIX does not work in portfiles, so emulate it
-if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore") # Windows
-    set(EXECUTABLE_SUFFIX ".exe")
-else()
-    set(EXECUTABLE_SUFFIX "")
-endif()
-
-if(EXISTS "${CURRENT_PACKAGES_DIR}/bin/ws${EXECUTABLE_SUFFIX}")
-    file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/tools/ixwebsocket")
-    file(RENAME "${CURRENT_PACKAGES_DIR}/bin/ws${EXECUTABLE_SUFFIX}" "${CURRENT_PACKAGES_DIR}/tools/ixwebsocket/ws${EXECUTABLE_SUFFIX}")
-    vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/ixwebsocket)
-endif()
-
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib/pkgconfig ${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig)
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
-
-if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
-    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin ${CURRENT_PACKAGES_DIR}/debug/bin)
-endif()
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
 
 # Handle copyright
 file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/ixwebsocket RENAME copyright)
-
-# Post-build test for cmake libraries
-# vcpkg_test_cmake(PACKAGE_NAME ixwebsocket)

--- a/ports/ixwebsocket/portfile.cmake
+++ b/ports/ixwebsocket/portfile.cmake
@@ -24,3 +24,6 @@ file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
 
 # Handle copyright
 file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/ixwebsocket RENAME copyright)
+
+# Post-build test for cmake libraries
+# vcpkg_test_cmake(PACKAGE_NAME ixwebsocket)

